### PR TITLE
Added GitHub Actions badge

### PR DESCRIPTION
**Pull request by radarhere**
Adds GitHub Actions badge after #169 added GitHub Actions

See https://github.com/radarhere/pillow-wheels/blob/badge/README.rst as a preview.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -9,6 +9,10 @@ This repository creates wheels for tagged versions of Pillow::
    :target: https://travis-ci.com/github/python-pillow/pillow-wheels
    :alt: Travis CI build status (wheels)
```
Comment (hugovk) [inline README.rst:10]:
```suggestion
   :alt: Travis CI build status
```

**Comment by hugovk** [comment]
It's not caused by this PR, but any idea why the macOS jobs are failing? There's a bunch of Python 2 warnings in there too.

```
  Uninstalling virtualenv-20.2.2:
    Successfully uninstalled virtualenv-20.2.2
  DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7. More details about Python 2 support in pip, can be found at pip.pypa.io/en/latest/development/release-process/#python-2-support
  WARNING: The directory '/Users/runner/Library/Caches/pip/http' or its parent directory is not owned by the current user and the cache has been disabled. Please check the permissions and owner of that directory. If executing pip with sudo, you may want sudo's -H flag.
  Uninstalling pip-19.3.1:
    Successfully uninstalled pip-19.3.1
  dyld: Library not loaded: /usr/local/opt/zstd/lib/libzstd.1.dylib
    Referenced from: /usr/local/opt/curl/bin/curl
    Reason: image not found
  multibuild/osx_utils.sh: line 277:  1676 Abort trap: 6           curl $MACPYTHON_URL/$py_stripped/${py_inst} > $inst_path
  Error: Process completed with exit code 134.
```

**Review by hugovk — APPROVED** [review]

**Comment by nulano** [comment]
@hugovk I'm looking into it right now. I see that the Python 2 warnings happened in successful runs as well (probably just need to change `pip` to `python -m pip`), but now `curl` is failing because it seems to depend on the brew `zstd` package which was causing conflicts with wheel building so it is removed at the start of the script.

**Comment by nulano** [comment]
I found a fix for the error (just need to remove curl from brew and use the system curl), but I'm a bit hesitant to open a PR. The Travis build in this repo takes around 1200 credits by my calculations which would make it the last arm build before the credits are sorted out. Is it possible to disable Travis on PRs here to buy a bit more time before the credits run out?

**Comment by hugovk** [comment]
Good idea!

You can add `[CI skip]` to commit messages, which disables Travis (and AppVeyor). GHA ignores it. But I think that'd still trigger a Travis build on merge.

For now, I've flipped these left two toggles off:

image

Let's remember to turn them back on for the release :)

I think I'll turn the others off too. Will comment on the main issue.

**Comment by nulano** [comment]
Oh right, I keep forgetting about `[ci skip]` because GHA doesn't support it. I added it to the commit message just to make sure Travis doesn't run and opened #173.

**Pull request merged**